cmd/frontend/internal/usagestats/search_latency.go

```diff
@@ -0,0 +1,92 @@
+package usagestats
+
+import (
+	"context"
+
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/db"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/types"
+)
+
+// GetSearchLatencyStatistics return search latency statistics for time spans of X days, Y weeks, and Z months respectively.
```

Comment:
A good practice for Go comments is to use the names of the parameters directly in comment prose. Ideally the names can be chosen to make that clear, as you've done here:

```suggestion
// GetSearchLatencyStatistics return search latency statistics for the specified span
// of days, weeks, and months respectively.
```

Reply:
(except "returns" instead of "return", which I missed in writing my suggestion)